client/components/filters/date/content.js

```diff
@@ -83,7 +83,7 @@ class DatePickerContent extends Component {
 						initialTabName={ 'custom' === period ? 'custom' : 'period' }
 						onSelect={ this.onTabSelect }
 					>
-						{ selectedTab => (
+						{ ( { name: selectedTab } ) => (
```

Comment:
Thanks for fixing this. Can we just use `name` instead of renaming it?

Reply:
I've switched to calling the arg `selected`, so we can do `selected.name` instead of `selectedTab` – I thought that just `name` wouldn't be clear.